react/week1/todo-app/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Todo from "./Todo";
+
+function App() {
+  return (
+    <div>
+      <h1>Todo-App</h1>
+
+      <Todo description="get out of bed" date="Wed Sep 13 2017" />
+      <Todo description="Brush teeth" date="Thu Sep 14 2017" />
+      <Todo description="Eat breakfast" date="Fri Sep 15 2017" />
```

Comment:
insted of hardcoding each todo in `app.js`, you could make a `TodoList` cpmponent that loops through an array of todos, and then use the `TodoList` conponent in`app.js`. It could look something like this:

```suggestion
<TodoList todos={todos} />
```

Reply:
Thanks Abdualrhman,.,I will replace the handcoded todo list with the array loop,.,